packages/esm-covid-app/src/views/patient chart/lab-results-schema-config.json

```diff
@@ -0,0 +1,197 @@
+        },
+        {
+          "name": "COVID Lab Test",
+          "excludedIntents": ["*"],
+          "uuid": "e92fe922-4863-11ec-99cc-1fdd2d4e9d88"
+        },
+        {
+          "name": "Lab Order Cancellation",
+          "excludedIntents": ["*", "COVID_LAB_CANCELLATION_EMBED"],
+          "uuid": "d38bc949-c95b-39eb-a2c5-08b82a36409c"
+        },
+        {
+          "name": "Sample Collection",
+          "excludedIntents": ["*", "COVID_SAMPLE_COLLECTION_EMBED"],
+          "uuid": "371d19b6-485f-11ec-99cc-1fdd2d4e9d88"
+        }
+      ]
+    },
+    {
+      "tabName": "Pending Lab Orders",
+      "headerTitle": "Pending Lab Orders",
+      "displayText": "Pending Lab Orders",
+      "encounterType": "a77d3e7f-5c8f-4074-a207-77a70e197b0c",
+      "columns": [
+        {
+          "id": "orderDate",
+          "isDate": true,
+          "title": "Date of Order",
+          "concept": "162078AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+        },
+        {
+          "id": "testType",
+          "title": "Test Type",
+          "concept": "069f6dfe-88c1-4a45-a894-0d99549c8718"
+        },
+        {
+          "id": "fowardLabreference",
+          "title": "Fowarded to Reference Lab",
+          "concept": "161934AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+        },
+        {
+          "id": "labStatus",
+          "title": "Status",
+          "concept": "6681366c-2174-489a-b951-13a1404935bf",
+          "type": "tag"
+        },
+        {
+          "id": "actions",
+          "title": "Actions",
+          "actionOptions": [
+            {
+              "formName": "COVID Lab Test",
+              "package": "covid",
+              "label": "View Details",
+              "mode": "view"
+            },
+            {
+              "formName": "Sample Collection",
+              "package": "covid",
+              "label": "Collect Sample",
+              "mode": "edit"
+            },
+            {
+              "formName": "COVID Lab Result Form",
+              "package": "covid",
+              "label": "Add/Edit Lab Result",
+              "mode": "edit"
+            }
+          ]
+        }
+      ],
+      "launchOptions": {
+        "displayText": "Add",
+        "moduleName": "@ohri/openmrs-esm-covid-app",
+        "hideFormLauncher": true
+      },
+      "formList": [
+        {
+          "name": "COVID Lab Result Form",
+          "uuid": "e92fe922-4863-11ec-99cc-1fdd2d4e9d88"
+        },
+        {
+          "name": "COVID Lab Result Form",
+          "uuid": "cf8cd756-baef-38df-b84d-92968b42c113"
+        },
+        {
+          "name": "Lab Order Cancellation",
+          "uuid": "d38bc949-c95b-39eb-a2c5-08b82a36409c"
+        },
+        {
+          "name": "Sample Collection",
+          "uuid": "371d19b6-485f-11ec-99cc-1fdd2d4e9d88"
+        },
+        {
+          "name": "COVID Lab Order Form",
+          "uuid": "f5fb6bc4-6fc3-3462-a191-2fff0896bab3"
+        }
```

Comment:
Thanks @hadijahkyampeire 
add this so that the form can open on view mode-->
```
        {
          "name": "COVID Lab Test",
          "uuid": "e92fe922-4863-11ec-99cc-1fdd2d4e9d88"
        }
```